gulpfile.js

```diff
@@ -31,8 +31,8 @@ const format = require('gulp-clang-format');
 
 const tsconfigPath = path.join(__dirname, 'tsconfig.json');
 const tslintPath = path.join(__dirname, 'tslint.json');
-const outDir = '.';
-const sources = ['src.ts/**/*.ts', 'src.ts/**/*.js'];
+const outDir = 'build';
+const sources = ['src.ts/**/*.ts'];
```

Comment:
If the output is now placed in `build/src`, we can rename `src.ts` back to `src`, but it could be better to do that in a follow-up.